vividus/src/main/java/org/vividus/transformer/SortingTableTransformer.java

```diff
@@ -34,6 +38,8 @@ public String transform(String tableAsString, TableParsers tableParsers, TablePr
     {
         TableRows tableRows = tableParsers.parseRows(tableAsString, properties);
         String byColumns = properties.getMandatoryNonBlankProperty("byColumns", String.class);
+        Order order = (Order) new FluentTrimmedEnumConverter().convertValue(Optional.ofNullable(properties
+                .getProperties().getProperty("order")).orElse(ASCENDING.toString()), Order.class);
```

Comment:
1) inject enum converter as a bean to transformer
2) avoid redundant conversions for default value: enum -> string -> enum

Reply:
done